Sources/Exporters/OpenTelemetryProtocol/metric/OltpHTTPMetricExporter.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import OpenTelemetrySdk
+import Foundation
+
+public func defaultOltpHTTPMetricsEndpoint() -> URL {
+    URL(string: "http://localhost:4318/v1/metrics")!
+}
+
+public class OtlpHttpMetricExporter: MetricExporter {
+    let endpoint: URL
+    let urlSession: URLSession // TODO do we need this?
```

Comment:
No, HTTPClient can be initialized with a given URLSession, so this is superfluous